### DISCUSSION TITLE
Unstable connexion with owon THS317-ET sensor

### DISCUSSION THREAD

**Discussion by bonjour81** [Koenkk/zigbee2mqtt#11373] [Devices]
Hi !
I’m trying to install an OWON THS317-ET temperature sensor.
I succeed in pairing but with difficulties and the sensor stops working after a while.

I have setup Z2M 1.23 (last docker available today), as the sensor was updated recently in Z2M.

So, here is what happened:

 sometime it won’t pair correctly.
 sometime it pair fine, but show only LQI & no more (no temp, no battery information).
 sometime it pair fine, send a few message with all data and then mute.
 sometime it pair fine, send a message every few minutes and then mute (works less than half an hour).

disconnect/reconnect battery might reanimate it, but no always.

Could it be a configuration / setup issue ? is there something I can try on Z2M ?

it may be also a sensor malfunction, but I want to make sure I tried all I can before send it back.

**Comment by 86nikko** [comment]
Hello @bonjour81,

Je viens d'acheter ce capteur de température OWON THS317-ET, je suis confronté aux mêmes problèmes que toi!
Pour ma part, je n'arrive pas à appairer correctement le capteur sous Z2M...
J'ai eu au mieux... le LQI...

As tu réussi a le faire fonctionner?

**Reply by bonjour81** [reply]
Bonjour @86nikko,
Non je l'ai renvoyé.
Je n'ai pas trouvé d'autre capteur zigbee équivalent (c'était pour une piscine), je pensais m'orienter vers un DS18B20 + ESP8266

Nope, I send it back.
I didn't found any alternative zigbee sensor (for a swimmingpool), so I'm considering to use a DS18B20 + ESP8266 instead.

**Reply by 86nikko** [reply]
Ah dommage... car ce capteur possède les bonnes caractéristiques et surtout "compatible" Z2M.
Moi aussi c'est exactement pour le même projet "piscine".

Je vais investiguer.
Merci pour ton retour
